### PR TITLE
Update READMEs and remove PNGs

### PR DESCRIPTION
**Pull request by tthyer**
* Remove architecture PNGs -- these are all at least partially inaccurate and won't be maintained anyway
* Update all READMEs, esp. adding a section on development

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,64 +1,49 @@
...
-
+ Read the [documentation](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/cfn-reference-servicecatalog.html) for more information on ServiceCatalog and CloudFormation.  
+
+Sage Bionetworks uses a second repository, [scipoolprod-infra](https://github.com/Sage-Bionetworks/scipoolprod-infra), to deploy Cloudformation stacks defined by the templates in this library.
+
+## Deployment
+Merges to master will copy these templates to an S3 bucket that is used by [scipoolprod-infra](https://github.com/Sage-Bionetworks/scipoolprod-infra) to deploy stacks.
+
+## Development
+Currently there is a manual process for development. The script `dev-deploy-templates.sh`, when run from the project root, takes a file or directory path and copys the templates found there to the deployment bucket. These can then be referenced when creating deployment stacks in [scipoolprod-infra](https://github.com/Sage-Bionetworks/scipoolprod-infra).
```
Comment (ahayden) [inline README.md]: One needs STS credentials derived from an IAM user in which account to run this?

Comment (tthyer) [reply README.md]: @ahayden I added a section about credentials and expanded on how stacks are deployed. Let me know if you think that is more clear?

File: README.md [hunk 2]
```diff
@@ -1,64 +1,56 @@
...
+
+### Account and Credentials
+While this library could be used in any AWS account, Sage Bionetworks is currently deploying to the "scipoolprod" account. In order to have permission to run the `dev-deploy-templates.sh` script, use the `AWSIAMCfServiceRole` role or something with similar permissions. Ask for guidance on this point from #sageit Slack channel.
+
+In the second half of the development process, when stacks are deployed from `scipoolprod-infra`, one needs access to the `AWSIAMAdminRole`.
```
Comment (zaro0508) [inline README.md]: this also uses the `AWSIAMCfServiceRole`

Comment (tthyer) [reply README.md]: No, not really. Remember this is specifically a _development_ guide. This has nothing to do with explaining how the CICD system works.

**Comment by tthyer** [comment]
@zaro0508 Thank you for the constructive comments about creating a central location for documenting everything about what we're doing with the Service Catalog. Good idea! Let's put the SC-108 documentation there too.

**Pull request merged**